lib/SearchAndSortPane/SearchAndSortPane.js

```diff
@@ -68,15 +68,17 @@ export class SearchAndSortPane extends Component {
       PropTypes.object,
     ]),
     visibleColumns: PropTypes.arrayOf(PropTypes.string).isRequired,
+    excludedSortColumns: PropTypes.arrayOf(PropTypes.string),
+    setInitialSortOnMount: PropTypes.bool,
     columnMapping: PropTypes.object.isRequired,
-    columnWidths: PropTypes.object.isRequired,
+    columnWidths: PropTypes.object,
```

Comment:
Do we need default value for not required prop?

Reply:
In this case, in my opinion, it does not make sense to provide as this property is [defined by the `MCLRenderer`](https://github.com/folio-org/stripes-components/blob/master/lib/MultiColumnList/MCLRenderer.js#L194).